msg/InternalResistanceEstimator.msg

```diff
@@ -0,0 +1,7 @@
+uint64 timestamp				# time since system start (microseconds)
+
+float32 internal_resistance_estimate            # [Ohm] Internal resistance per cell estimate
+float32 voc_estimate                            # [V] VoC estimate
```

Comment:
VoC? Cell voltage corrected?

Reply:
I now consistently named everything OCV (Open circuit voltage), which is the term that is used in all the literature.